### ISSUE TITLE
Support custom repositories with authentication?

### ISSUE THREAD

**Issue by Hawods** [tarn2206/gradle-dependencies-checker#10]

**Comment by emongstad** [comment]
Yes, this would be a great improvement. We're using Artifactory with basic user/password authentication. The plugin marks "unauthorized" to all local dependencies in the Gradle Dependency Updates-view
